src/assets/styles/_header.scss

```diff
@@ -11,7 +11,7 @@
 .heading {
   padding: 0;
   margin: 0;
-  font-weight: var(--calcite-ui-text-weight-demi);
+  font-weight: var(--calcite-font-weight-medium);
```

Comment:
I think you can `@apply font-medium` here, perhaps?

Reply:
hmm, wait maybe not? I'm not sure how the shared assets styles work...

Reply:
@paulcpederson Totes. I should have noted that I plan to start working through other components and partials once this PR is in approved form.

The Block refactor is in there as an example of how the rest of stuff will be refactored.